scripts/common/kubernetes.sh

```diff
@@ -61,7 +61,11 @@ function kubernetes_sysctl_config() {
 function kubernetes_install_host_packages() {
     k8sVersion=$1
 
-    logStep "Install kubelet, kubeadm, kubectl and cni host packages"
+    # ALWAYS update crictl: https://listman.redhat.com/archives/rhsa-announce/2019-October/msg00038.html 
```

Comment:
1. This won't work before `kubernetes_get_host_packages_online` for online installs
2. We only need to upgrade if they have an outdated crictl.

Reply:
Moved this after we grab packages. I didn't necessarily check the crictl version since we are always syncing what's in the bundle with the system if any one component is missing. Let me know if this doesn't resolve this comment.